.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
@ronanguilloux See this build: https://travis-ci.org/Soullivaneuh/IsoCodesValidator

This project has `fast_finish` option enabled.

With this option, you have two concrete cases:

 * One of required jobs failed => Mark Travis build **failed** even if another jobs are not finished
 * All required jobs failed and one optional jobs (allow_failures) failed => Mark Travis build **succeed** even if another jobs are not finished

In any case, all the jobs will be completely finished, no changes about it.

For example, this could be useful to see more quickly if PR are good to merge, or not, according to Travis.